bin/commands/disable.js

```diff
@@ -1,12 +1,31 @@
+/* eslint-disable no-console */
```

Comment:
This may be the right time to implement KUZ-632 (removal of service enabling/disabling features).
What do you think?